edb/server/compiler/explain.py

```diff
@@ -48,6 +48,7 @@
     rf'(\.?"?({uuid_core})"?)',
     re.I
 )
+case_re = re.compile(r'(?<!^)(?<!\s|-)[\s-]*(?=[A-Z])')
```

Comment:
Could you add a comment explaining this regex, since I don't understand it at a glance

Reply:
Added some reasoning. If anyone wants to piece-by-piece description, AI does that well :)